### PR TITLE
FE-456 | Add contributing md

### PR DESCRIPTION
**Pull request by fauna-ryanharris**
### Notes
[JIRA Ticket](https://faunadb.atlassian.net/browse/FE-456)

### CODE REVIEW

**Comment by BrunoQuaresma** [comment]
Do we want to enforce the commit lint style?

**Comment by fauna-ryanharris** [comment]
> Do we want to enforce the commit lint style?

@BrunoQuaresma Do you mean the `commitizen` formatting and auto-generated changelog?

**Comment by BrunoQuaresma** [comment]
> > Do we want to enforce the commit lint style?
> 
> @BrunoQuaresma Do you mean the `commitizen` formatting and auto-generated changelog?

yeap.

**Comment by fauna-ryanharris** [comment]
@BrunoQuaresma Agreed. I think we should enforce those things going forward, but we'll need to setup some tooling to handle it 🚀

**Comment by fauna-ryanharris** [comment]
@danieltodonnell I'll hold off on merging this until you weigh in 👍

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+### Pull Request Guidelines
+
+When issuing a PR, please do the following to help us expedite merging your work:
+
+- Add a description to your PR
+
+- Update the `README.md` file to describe your new functionality
+
+- Ensure [maintainers can make changes](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to your PR
```
Comment (danieltodonnell) [inline CONTRIBUTING.md]: Can you make this first and preface it with an **IMPORTANT:**

**Pull request merged**